src/electron/views/device-connect-view/components/android-setup/android-setup-step-component-provider.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { CommonAndroidSetupStepProps } from 'electron/views/device-connect-view/components/android-setup/android-setup-types';
+import * as React from 'react';
+
+export class AndroidSetupStepComponent extends React.Component<CommonAndroidSetupStepProps> {
```

Comment:
I think this class name might be a bit confusing; I feel like it implies that it should represent one component of an android setup step, rather than the thing responsible for selecting the appropriate component based on input. `AndroidSetupStepDispatcher`? `CurrentAndroidSetupStep`? `AndroidSetupStepContainer`? `AndroidSetupWizard`?